vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -75,99 +77,129 @@ public ImageData getImageDimensions(IVSImage image, int width, int height) {
         int imageHeight = image.getHeight();
 
         Float newWidth = (new Float(height) / new Float(imageHeight)) * new Float(imageWidth);
-        if (newWidth < width) {
+        if(newWidth<width) {
             return new ImageData(height, newWidth.intValue());
         }
 
         Float newHeight = (new Float(width) / new Float(imageWidth)) * new Float(imageHeight);
         return new ImageData(newHeight.intValue(), width);
     }
-    
+
     /**
-     * Method to return the requested images  
+     * Method to return the requested images
      * 
-     * @param pageNo. if pageNo<1, 1st page is returned, if pageNo>total pages,last page is returned
+     * @param pageNo. if pageNo<1, 1st page is returned, if pageNo>total pages,last
+     *                page is returned
      * @return list of images in the requested pageNo
-     */ 
+     */
     @Override
-    public List<IVSImage > getImages(int pageNo) {
+    public List<IVSImage> getImages(int pageNo) {
```

Comment:
this method does the same thing as below `getImages` method, doesn't it? except using default values? it should call the other method to avoid duplicate code.